test/resources/title-with-dashes-form.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml">
+  <h:head>
+    <h:title>this-title-has-dashes</h:title>
+    <model>
+      <instance>
+        <data id="this-id-has-dashes">
```

Comment:
Let’s make the world a better place by not saying dash when we mean hyphen.
- hyphen
– en dash
— em dash
- your thing. Yup, it’s a hyphen. :-)

Reply:
TIL that dashes are not hyphens :D